openedx_events/content_authoring/data.py

```diff
@@ -65,10 +78,12 @@ class XBlockData:
     Arguments:
         usage_key (UsageKey): identifier of the XBlock object.
         block_type (str): type of block.
+        version (UsageKey): identifier of the XBlock object with branch and version data (optional).
     """
 
     usage_key = attr.ib(type=UsageKey)
     block_type = attr.ib(type=str)
+    version = attr.ib(type=UsageKey, default=None, kw_only=True)
```

Comment:
Although this change is backward compatible, I'd like to know why are we doing it?